parser/src/operators.rs

```diff
@@ -124,7 +124,7 @@ impl Operator for BinaryOperator {
 			BinaryOperator::LessThan => "<",
 			BinaryOperator::GreaterThan => ">",
 			BinaryOperator::LessThanEqual => "<=",
-			BinaryOperator::GreaterThanEqual => ">",
+			BinaryOperator::GreaterThanEqual => ">=",
```

Comment:
This one was catched by clippy as it was initially a duplicated pattern. I assumed it was a bug a fixed it